modules/loader-utils/src/lib/files/node-file-facade.ts

```diff
@@ -35,6 +35,16 @@ export class NodeFileFacade implements ReadableFile, WritableFile {
   async stat(): Promise<Stat> {
     throw NOT_IMPLEMENTED;
   }
+
+  /** Truncates the file descriptor. */
```

Comment:
Since these new methods are Node file specific (i.e. not implemented on WritableFile, BlobFile etc, maybe list them at the end in a specially marked section?